src/core/data/server.c

```diff
@@ -53,6 +61,24 @@ _server_close(struct buf_sock *s)
     buf_sock_return(&s);
 }
 
+#ifdef USE_EVENT_FD
+static inline void
+_server_notify_worker(void)
```

Comment:
How about absorbing _server_pipe_write() into this new function, so the details of eventfd va pipe are concentrated within two functions at most?